.github/workflows/main.yml

```
@@ -0,0 +1,43 @@
+# Taken from https://github.com/ssato/python-anyconfig/blob/next/.github/workflows/tests.yml
+# https://github.com/ssato/python-anyconfig/blob/next/LICENSE.MIT
+name: make
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build:
+    runs-on: ${{ matrix.platform }}
+    strategy:
+      max-parallel: 5
+      matrix:
+        platform:
+          - ubuntu-latest
+        python-version:
+          - 3.6
```

```suggestion
          - 2.7
          - 3.4
```

(skipping 3.5 and 3.6 here to keep CI smaller)

---

I don't know well about python's ecosystem.
As far as trying locally, 3.5 is not available.
So I guessed ones older than 3.5 were not available.
How about the other versions later?

---

Those should be available from the GitHub action.